src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -1,11 +1,32 @@
 package core.basesyntax.service;
 
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
 
+    private static final StorageDao storageDao = new StorageDaoImpl();
+
     @Override
     public User register(User user) {
-        return null;
+        if (user == null || user.getId() == null || user.getLogin() == null
+                || user.getPassword() == null || user.getAge() == null) {
+            throw new RuntimeException("Please provide valid registration data");
+        }
```

Comment:
i think better split this if block and throw more informative exception for invalid login, password, age